### DISCUSSION TITLE
What are WWWWW?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#729] [Galaxy Trivia] [answered]
**Bonus:** What professions use this "technique"?

**Comment by peterbaccaro** [comment, marked as answer]
WWWWW can refer to the five W's (**Who, What, When, Where, Why**), which are a set of questions used in **journalism, research, and other fields to gather information about a topic**. The five W's are Who (referring to the person or people involved), What (the subject matter), When (the time frame), Where (the location), and Why (the reason or motivation).
